third_party/pytest/defs.bzl

```diff
@@ -0,0 +1,22 @@
+load("@rules_python//python:defs.bzl", "py_test")
+load("@rules_ros2_pip_deps//:requirements.bzl", "requirement")
+
+def py_pytest_test(name, srcs, deps = [], args = [], **kwargs):
```

Comment:
For default args, it's a good practice to use immutable args, best `None` --  here I would use None, and then within the macro `deps = deps or []`.